support-pie/donate.html

```diff
@@ -52,19 +52,20 @@
 				<li>Tools available to teams when working on the UC Berkeley campus</li>
 			</ul>
 
-			<p>We ask teams for only a $100 entry fee in order to ensure that money is not a barrier to entry. Our budget is extremely efficient and cost-effective compared to most robotics competitions, which often require each team to spend thousands of dollars.</p>
+			<p>We ask teams for only a $100 entry fee in order to ensure that money is not a barrier to entry. Our budget is extremely efficient and cost-effective compared to most robotics competitions, which often require each team to spend thousands of dollars. Over $900 of our kit is subsidized by Pioneers in Engineering.</p>
+      <img src="../assets/sponsors/kit-cost.png" style="width:100%">
 		</div>
 
 		<div class="col-md-6">
-			<h3>PREP</h3>
+			<h3>Workshops</h3>
 
-			<p>Our PREP program provides weekly engaging, hands-on learning experiences to students in two high schools. We fund all materials that we use for these lessons. Your donations let us provide:</p>
+			<p>We hold a number of workshops on and off Berkeley's campus, for high school students and Berkeley students alike. Our workshops are always free for participants, so funding comes out of our own budget. Your donations help us provide:</p>
 
 			<ul class="donations">
 				<li>Trebuchet construction materials</li>
-				<li>Arduino microcontrollers to control soda-fetching robots</li>
-				<li>LEDs to create music visualizers</li>
-				<li>Snap circuits to build transistor radios</li>
+				<li>Materials to fabricate a plywood gear kit for 90+ students</li>
```

Comment:
Can you add a bullet point at the very top that more directly talks about the workshop? For example,
- [2] free workshops per semester for [30+ high school] students (or whatever the data/plan is)
- Workshops cover software, mechanical engineering (i.e. more specific details)
- Workshops at the schools themselves?
And also maybe feature testimonies or feedback results from students, i.e. 70% satisfaction or change in career plans or something